src/WarcraftLegacies.Source/Quests/Scourge/QuestEnemyEncroachment.cs

```diff
@@ -0,0 +1,34 @@
+﻿using MacroTools.FactionSystem;
+using MacroTools.ObjectiveSystem.Objectives.UnitBased;
+using MacroTools.QuestSystem;
+
+namespace WarcraftLegacies.Source.Quests.Scourge
+{
+  public sealed class QuestEnemyEncroachment : QuestData
+  {
+    
+    /// <inheritdoc />
+    public QuestEnemyEncroachment(Faction faction) : base("Enemy Encroachment", "Hostile forces have stepped foot onto Northrend! The Frozen Throne will have to employ new powers to defend against these interlopers", @"ReplaceableTextures\CommandButtons\BTNAnimateDead.blp")
```

Comment:
The Description field should be present tense; the player can read this before hostile forces have stepped foot onto Northrend so it isn't quite true. Maybe something like...

```suggestion
    public QuestEnemyEncroachment(Faction faction) : base("The Slumbering King", "Ner'zhul commands the undead hordes from his throne atop Icecrown, waiting patiently for the inevitable day that interlopers will come to invade his frozen lands.", @"ReplaceableTextures\CommandButtons\BTNAnimateDead.blp")
```